cicd/docker/algod/entrypoint.sh

```diff
@@ -0,0 +1,28 @@
+set -e
+
+if [ $UPDATE_ON_START = true ]; then
+  if [ -z ${ALGOD_VERSION} ]; then
+    python3 /algod/setup.py install \
+      --bin-dir "$HOME/node" \
+      --channel "${CHANNEL}"
+  else
+    python3 /algod/setup.py install \
+      --bin-dir "$HOME/node" \
+      --channel "${CHANNEL}"
+      --algod-version ${ALGOD_VERSION}
+  fi
+fi
+
+mkdir -p /opt/testnetwork
+
+python3 /algod/setup.py configure-network \
```

Comment:
This should be done at build time, why reconfigure the container every time it starts?

Reply:
This is so that if things like algod/kmd port or network token needs to be changed at start time. There can be instances where this is useful like times when we may want to run several of these on one kubernetes pod with shared networking